96284658_nishthakumari/assignment2

```diff
@@ -0,0 +1,57 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+
+
+
+
+
+__global__ void MatrixAdd(float *a, float *b, float *c, int M,int N) {
+
+   int col = threadIdx.x + blockIdx.x * blockDim.x;
+   int row = threadIdx.y + blockIdx.y * blockDim.y;
+   int index = col + row * N;
+   if (col < N && row < M) {
+       c[index] = a[index] + b[index];
+   }
+}
+
+
+
+int main(){
+
+  my_mat *A, *B, *C;
```

Comment:
MyMat is a library in python and I don't think it is present in C. You need to declare them as float pointers.